test/test_conditional_permutation_importance.py

```diff
@@ -0,0 +1,570 @@
+from copy import deepcopy
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.exceptions import NotFittedError
+from sklearn.linear_model import LinearRegression, LogisticRegression
+from sklearn.metrics import log_loss
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import root_mean_squared_error
+
+from hidimstat import CPI, BasePerturbation
+from hidimstat._utils.exception import InternalError
+
+
+def run_categorial_cpi(X, y, n_permutation, seed):
```

Comment:
why 'categorical' ?

Reply:
It's an old part of the code. I don't remember it.
I remove it.